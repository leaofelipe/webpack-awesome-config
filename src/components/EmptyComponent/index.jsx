import React from 'react'

export default () => <div>Empty Component.</div>
