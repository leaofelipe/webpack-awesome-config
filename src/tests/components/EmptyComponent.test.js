import React from 'react'
import { shallow } from 'enzyme'
import EmptyComponent from '../../components/EmptyComponent'

describe('Empty Component', () => {
  test('should be rendered correctly', () => {
    const wrapper = shallow(<EmptyComponent />)
    expect(wrapper).toMatchSnapshot()
  })
})
