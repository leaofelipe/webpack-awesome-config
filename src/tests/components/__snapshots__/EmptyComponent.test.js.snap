// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty Component should be rendered correctly 1`] = `
<div>
  Empty Component.
</div>
`;
